libraries/core_mqtt_demo_dependencies.cmake

```diff
@@ -1,6 +1,6 @@
 afr_module(NAME core_mqtt )
 
-afr_set_lib_metadata(ID "core_mqtt")
+afr_set_lib_metadata(ID "core_mqtt_demo_dependencies")
```

Comment:
This should be core_mqtt?

Reply:
We will not be adding any metadata for `core_mqtt` but only for the dependencies target